zetaclient/chains/solana/contract/gateway.json

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
It seems to me that it would make more sense to add an independent package in `pkg` for the gateway usage

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/eb814a91d89bee5be48174081af7338f76cb197a